docs/docs/tutorial/part-1/index.mdx

```diff
@@ -88,6 +88,17 @@ gatsby new
 
 <Announcement style={{marginBottom: "1.5rem"}}>
 
+You can also generate a Gatsby site without needing to go through all the prompts. In this case, the the defaults will be used except for the site name which you'll need to pass.
+You can do this by running the following command from the command line:
+
+```shell:title=Desktop
+ gatsby new <site-name> -y
```

Comment:
There is no need for a flag here, `gatsby new`'s interface is different from `npm init gatsby` and `npx create-gatsby`.

Calling [`gatsby new` with a site name](https://www.gatsbyjs.com/docs/reference/gatsby-cli/#new) uses the default starter and skips the questionnaire. Calling `gatsby new` uses the questionnaire.

Reply:
Thanks! As we earlier discussed, I initially thought we pass the `-y` flag straight into `create-gatsby`